tutorials/privacy/mempool-space/tutorial.yml

```
@@ -0,0 +1,21 @@
+id: f3e468a1-92f1-43ce-b2e4-c3298fa0e02f
+project_id: d1918770-fe6a-44aa-b639-977e9ece2450
+professor_id: 81a82a0f-3003-4459-9d6a-67f860efb20e
+level: beginner
+tags:
+  - privacy
+  - analysis
+  - mempool
```

```suggestion
  - onchain
```
Be careful, we should only use the tags from Paolo’s list: "mempool" and "analysis" are not on it.
https://github.com/PlanB-Network/bitcoin-educational-content/blob/dev/docs/50-planb-tags.md